lib/rules/forbid-prop-types.js

```diff
@@ -13,6 +13,7 @@ const propWrapperUtil = require('../util/propWrapper');
 // Constants
 // ------------------------------------------------------------------------------
 
+const REACT_MODULE = 'react';
```

Comment:
This needs to come from the pragma, so it can be overridden - not hardcoded to "React".

Reply:
Sorry, I'm not sure I understand this.

From what I can see, the pragma util refers to instances where the code refers to an imported `React`, for example `React.Fragment`. I was using the example below as a point of reference:

https://github.com/yannickcr/eslint-plugin-react/blob/4a72e6a0784f7cc7f16f4574ce7f14de55540a73/lib/rules/jsx-fragments.js#L97-L122


I'm using this `REACT_MODULE` constant to determine when something is _being imported_ from React, for example `import { PropTypes } from React`. Do I need to make changes to the pragma util in order to support this? Or is there something else I've missed here?

Thanks for your help with this PR, so far 🙂

Reply:
ah, i see what you're saying. I suppose this is fine as-is, then.